security-advisories/mbedtls-security-advisory-2023-10-1.md

```diff
@@ -0,0 +1,39 @@
+# Buffer overread in TLS stream cipher suites
+
+**Title** |  Buffer overread in TLS stream cipher suites
+---|---
+**CVE** |  CVE-2023-43615
+**Date** |  05 October 2023
+**Affects** |  All versions of Mbed TLS
+**Impact** |  A remote attacker may cause a crash or information disclosure.
+**Severity** |  Medium
```

Comment:
Why are we issuing a security advisory for a medium severity issue?

Reply:
We discussed the severity in a security team meeting IIRC. An unauthenticated peer can crash a TLS server on a typical platform, so we decided to issue a CVE.

Reply:
Yes, I remember discussing it on slack, I think we agreed that it was on the top end of medium. My memories about the CVE are more blurry.

In the past, issuing a CVE meant that we have to publish an advisory on the website as well. This is not the case anymore, our new process doesn't mandate this. We can, if we want, but we don't have to.